ocaml/xapi/xapi_pvs_cache_vdi.ml

```diff
@@ -0,0 +1,56 @@
+(*
+ * Copyright (C) 2016 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Client
+
+let size = Int64.of_int (1024 * 1024 * 1024) (* TODO: how big? *)
```

Comment:
We probably need to add a way to configure the size through the API somehow. We can leave that to later.